README.md

```diff
@@ -1,2 +1,63 @@
-# helm-charts
-Iterative helm charts
+## Studio Helm Chart
+
+## Install Studio
+
+Configure Credentials to pull images from secure registry
+```bash
+$ kubectl create secret docker-registry iterativeai \
+    --docker-server=docker.iterative.ai \
+    --docker-username=username \
+    --docker-password=password
+```
+
+Create a file `override.yaml` with the following content or edit `values.yaml`
+
+```yaml
+imagePullSecrets:
+  - name: iterativeai
+```
+
+ Deploy Studio
+ ```bash
+$ helm install studio studio/ -n <namespace> -f override.yaml
```

Comment:
Please add command to update dependencies

```suggestion
$ helm dependency update
$ helm install studio studio/ -n <namespace> -f override.yaml
```

In another commenet I asked to  remove `charts/` directory. To have it working user needs to build dependencies on own, based on `Chart.lock`.

Reply:
Updated README as requested